ecl/dedicated_hypervisor/v1/_proxy.py

```diff
@@ -264,3 +265,62 @@ def update_cfgw_connection(self, server_id):
         """
         cfgw = _server.CFGWConnection()
         return cfgw.update_cfgw_connection(self.session, server_id)
+
+    def show_vcenter(self):
+        """
+        Shows the registered vCenter server information.
+
+        :return: One :class:`~ecl.dedicated_hypervisor.v1.vcenter.show_vcenter`
+        instance.
+        """
+
+        return _vcenter.Vcenter.show_vcenter(self.session)
+
+    def register_vcenter(self, password, license_id):
+        """
+        Register the vCenter Server.
+
+        :param password: The password of vCenter server. Available character is
+                1-20 character of alphabet[a-zA-Z], number[0-9] and Symbols[
+                .-_/*+,!#$%&()~|].
+        :param string license_id: ID for license of vCenter server.
+
+        :return: One :class:`~ecl.dedicated_hypervisor.v1.vcenter.register_vcenter`
+        instance.
+        """
+
+        body = {}
+        body["password"] = password
+        body["license_id"] = license_id
```

Comment:
```suggestion
        body = {
            "password": password
            "license_id": license_id
        }
```
password と license_id を必ず body にセットするなら、bodyの初期化で password と license_id をセットした方が良いです（body = {} は無駄な処理です）。

Reply:
ご指摘の箇所を削除しました。